File: pset6/mario/mario.py
from cs50 import get_int

while True:
    # prompt user
    height = get_int("Height: ")

    # Make sure the number is more then 0 and less then 24
    if height < 24 and height >= 0:
        break;

# Print
for i in range(height):
    # First part
    for j in range(height - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")

    # Middle two spaces
    for l in range(2):
        print(" ", end="")

    # End part
    for m in range(i + 1):
        print("#", end="")

    print("\n", end="")

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");

        int numberOfSpace = height - 1;
        if (height < 9 && height > 0)
        {
            for (int i = 0; i < height; i++)
            {
                for (int j = 0; j < numberOfSpace; j++)
                {
                    printf(" ");
                }

                for (int k = 0; k < height - numberOfSpace; k++)
                {
                    printf("#");
                }

                printf("\n");
                numberOfSpace--;
            }
            break;
        }
    } while (true);
}
